docs/fork_choice.md

```diff
@@ -57,17 +57,17 @@ $$W_N = B_N + \sum_i^{i \in \text{children}[N]}W_i$$
 
 In the previous scheme, there are two rewards:
 
-- Proposer rewards, given to a proposer when their block is included in the chain. This also adds an incentive for them to try to predict the most-likely branch to be the canonical one.
-- Attester rewards, which are smaller. These are given if the blocks they attest to are included.
+- Proposer rewards, are given to a proposer when their block is included in the chain. This also adds an incentive for them to try to predict the most likely branch to be the canonical one.
```

Comment:
If we introduce "are" here, let's remove the `,` after rewards.

Reply:
Sure. I removed it in #778